insights/combiners/cpu_vulns_branch.py

```diff
@@ -0,0 +1,34 @@
+"""
+CpuVulnsBranch - combiner for cpu vulnerabilities:
```

Comment:
Why is the name `CpuVulnsBranch`?

Reply:
no much good name to me, may I have your advice on this name?

Reply:
Maybe the same name as for the parser? `CpuVulns` would not make a naming conflict as they are both in different modules. Or maybe `CpuVulnsCombined` or `CpuVulnsAll` as there is `HttpdConfAll` which has similar function.

Reply:
CpuVulnsCombined sound good to me, let me modify it. Thanks :)

Reply:
@skontar finally, I updated the combiner name with CpuVulnsAll. will you please review this PR again? Thanks!